src/components/Badge/Badge.scss

```diff
@@ -1,13 +1,13 @@
 @import '../../styles/common';
 
-$height: rem(20px);
-$small-height: rem(18px);
-$extra-small-height: rem(16px);
+$height: spacing(base);
```

Comment:
Using spacing function because the we currently only bring in the color values from `@shopify/polaris-tokens` 😑 We're gonna be changing the spacing system in the near future, so not bothering to import the spacing tokens into `src/utilities/theme/tokens.ts` or `src/utilities/custom-properties`.